guides/assets/asset-management.md

```diff
@@ -0,0 +1,161 @@
+
+- `app/assets`
+- `lib/assets`
+- `vendor/assets`
+
+Asset trees are divided into subdirectories according to their types: either
+`images`, `javascripts`, or `stylesheets`.
+
+Solidus takes this further by scoping each asset type by `spree/frontend` or
+`spree/backend`, depending on where the asset is being used.
+
+The structure for the `app` and `vendor` trees looks like this:
+
+```
+app|vendor
+|-- assets
+    |-- images
+    |   |-- spree
+    |       |-- frontend
+    |       |-- backend
+    |-- javascripts
+    |   |-- spree
+    |       |-- frontend
+    |       |-- backend
+    |-- stylesheets
+        |-- spree
+            |-- frontend
+            |-- backend
+```
+
+This directory structure is designed to keep assets from the `solidus_frontend`
+and `solidus_backend` from conflicting with each other.
+
+Solidus also generates top-level [manifests][rails-manifests] that require all
+of the Solidus-provided stylesheets and JavaScript files as well as your own
+site-specific files.
+
+To see the stock Solidus assets, you can check the contents of the
+`solidus_frontend` and `solidus_backend` gems installed on your system or
+[the `app/assets` contents in the Solidus GitHub repo][solidus-assets-contents].
+
+[rails-assets-pipeline]: http://guides.rubyonrails.org/asset_pipeline.html
+[rails-manifests]: http://guides.rubyonrails.org/asset_pipeline.html#manifest-files-and-directives
+[solidus-assets-contents]: https://github.com/solidusio/solidus/tree/master/frontend/app/assets
+
+## Solidus manifests
+
+The `solidus_frontend` and `solidus_backend` gems provide [asset
+manifests][rails-manifests] that bundle up all the JavaScript files and stylesheets
+that they require. For example, see the `all.css` and `all.js` manifests in the
+`vendor` tree:
+
+```
+vendor
+|-- assets
+    |-- javascripts
+    |   |-- spree
+    |       |-- frontend
+    |       |   |-- all.js
+    |       |-- backend
+    |           |-- all.js
+    |-- stylesheets
+        |-- spree
+            |-- frontend
+            |   |-- all.css
+            |-- backend
+                |-- all.css
+```
+
+Your project's `vendor/assets/javascripts/spree/backend/all.js` file would show
+you that your Solidus backend include jQuery and any other files that you create
+in this `spree/backend` directory:
+
+```javascript
+//= require jquery
+//= require jquery_ujs
+//= require spree/backend
+//= require_tree .
+```
+
+<!-- TODO:
+  Because a typical Solidus installation includes a few manifest files – and
+  not all of them have the same name – it might be worthwhile to extend the
+  documentation of them.
+-->
+
+## Managing application assets
+
+We recommend using the [Rails's suggested approach to asset
+organization][asset-organization], then scoping your custom files to the
+`spree/frontend` or `spree/backend` subdirectories to avoid conflicts or
+accidental file overrides.
+
+For example, if you want to use the [Foundation CSS framework][foundation] in
```

Comment:
Missing link

Reply:
I added this at the bottom of the section. Thank you.